cli/command/network/connect.go

```diff
@@ -38,11 +38,11 @@ func newConnectCommand(dockerCli *command.DockerCli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVar(&opts.ipaddress, "ip", "", "IP Address")
-	flags.StringVar(&opts.ipv6address, "ip6", "", "IPv6 Address")
+	flags.StringVar(&opts.ipaddress, "ip", "", "Container IPv4 address (e.g. 172.30.100.104)")
+	flags.StringVar(&opts.ipv6address, "ip6", "", "Container IPv6 address (e.g. 2001:db8::33)")
```

Comment:
I think it is good to have the exact same help message for these options in docker run and docker network connect, because the options are really equivalent.
IMO, I would drop the word "container" in both the CLIs. Because as you say it may sound redundant and also because somebody could argue that that is the IP address of one of the container's interfaces, not the container's only address.